src/corg/development-tools-and-process/git-usage.md

```diff
@@ -730,7 +730,7 @@ Recursively executes the given Shell command on the current and its direct child
     * When your work is achieved, rebase then merge the branch (see [Isolated testing with ondemand-testandpush jobs]({{page page='isolated-testing'}})). Optionally delete it.
 * Use the same name for a local branch than its corresponding remote branch.
 * Always reference JIRA issues in the commit comments.
-* Commit as a valid author (i.e. "Firstname Lastname <address@somewhere.com>" with "address@somewhere.com" being configured in your GitHub account).
+* Commit as a valid author (i.e. "Firstname Lastname \<email address>" with the email address being configured in your GitHub account, no matter if it's public or private).
```

Comment:
You added a black slash before the email address, does it aim to escape the left chevron?